cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/GateChangeEndpointInput.java

```diff
@@ -105,11 +103,11 @@ public static final class Post extends EndpointInput {
         static final boolean DEFAULT_FAIL_IF_EXISTS = true;
         static final String FAIL_IF_EXISTS = "fail-if-exists";
         private boolean failIfExists = true;
-        private final Set<GateChange> changes = new HashSet<>();
+        private final Set<GateChange> changes;
 
         private Post(Set<GateChange> changes) {
             super();
-            changes.addAll(Objects.requireNonNull(changes, "Set of changes is required to be non-null"));
+            this.changes = Objects.requireNonNull(changes, "Set of changes is required to be non-null");
```

Comment:
Why did you change this?

Reply:
Changes were not being added to the set correctly. The result of attempting to add values was an empty set.

Reply:
Fixed in latest push